.github/workflows/test.yml

```diff
@@ -35,3 +35,6 @@ jobs:
       - name: Lint with pylint
         run: |
           pylint examples pychromecast
+      - name: Lint with rstcheck
+        run: |
+          rstcheck README.rst
```

Comment:
Maybe rewrite it in markdown instead? Almost all our readmes use markdown.

Reply:
Yeah, I think that makes sense, we can do that in a follow-up.